e2e/startup-profiles.e2e.spec.ts

```diff
@@ -171,6 +173,31 @@ test.describe.serial('track startup windows based on existing profiles and setti
       expect(contents).toMatch(errorMatcher);
     });
 
+    test('missing version', async() => {
+      const filename = `${ __filename }-missing-settings-version`;
+      const logDir = reportAsset(filename, 'log');
+      const logPath = path.join(logDir, 'background.log');
+      const versionLessSettings: RecursivePartial<Settings> = {
+        kubernetes:  { enabled: true },
+        application: {
+          debug:                  true,
+          pathManagementStrategy: PathManagementStrategy.Manual,
+          startInBackground:      false,
+        },
+      };
+      const settingsFullPath = path.join(paths.config, 'settings.json');
+
+      fs.mkdirSync(paths.config, { recursive: true });
+      fs.writeFileSync(settingsFullPath, JSON.stringify(versionLessSettings));
```

Comment:
You're in an `async` function, prefer the promise version over sync.

Reply:
yup